src/entity/SourceMember.ts

```diff
@@ -54,7 +50,3 @@ export class SourceMember {
   @Column({ type: 'jsonb', default: {} })
   flags: SourceMemberFlags;
 }
-
-const TOKEN_BYTES = 32;
-export const generateMemberToken = async (): Promise<string> =>
```

Comment:
Removed this in favour of `crypto.randomUUID`, its faster, sync and more simple and I don't think we get anything from using custom alphabet.